Model/Api/ShippingMethods.php

```diff
@@ -252,7 +251,7 @@ protected function checkCartItems($cart)
         foreach ($this->quote->getAllVisibleItems() as $item) {
             $sku = trim($item->getSku());
             $quantity = round($item->getQty());
-            $unitPrice = $item->getCalculationPrice();
+            $unitPrice = round($item->getCalculationPrice(), 2);
```

Comment:
@daisy1754 is this rounding appropriate, or should it be based on the precision map from your previous PR?

Reply:
good call; i was planning on refactoring all of such code once this and my other one is merged

Reply:
Understood 👍